packages/web/src/store/tipping/utils.ts

```diff
@@ -0,0 +1,187 @@
+import { ID } from 'common/models/Identifiers'
+import { UserTip } from 'common/models/Tipping'
+import { Nullable } from 'common/utils/typeUtils'
+import { FEED_TIP_DISMISSAL_TIME_LIMIT } from 'utils/constants'
+
+const RECENT_TIPS_KEY = 'recent-tips'
+
+type RecentTipsStorage = {
+  dismissed: boolean
+  minSlot: number
+  lastDismissalTimestamp: Nullable<number>
+}
+
+const getRecentTipsStorage = () => {
+  if (!window.localStorage) {
+    return null
+  }
+
+  const value = window.localStorage.getItem(RECENT_TIPS_KEY)
+  if (!value) {
+    return null
+  }
+
+  return JSON.parse(value) as RecentTipsStorage
+}
+
+export const getMinSlotForRecentTips = () => {
+  const storage = getRecentTipsStorage()
+  return storage ? storage.minSlot : null
+}
+
+export const getDismissedSlotForRecentTips = () => {
+  const storage = getRecentTipsStorage()
+  return storage ? storage.dismissed : null
+}
+
+export const checkRecentTip = ({
```

Comment:
nit: naming feels a bit unclear, could be something like `findRecentTipToDisplay`?